nss-day-cohort-45/nutyshell-kicking-the-seat-1
# Jh public message

# Description
Styled message box and added delete buttons according to active user.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
1. Login
2. Send message and look for delete button.
3. Login to second account.
4. Send another message, should only be one delete and 2 messages
5. Delete message.
6. Booba
# Checklist:
